README.md

```diff
@@ -2,42 +2,1104 @@
+
+## Mapping in Blindata
+
+The observer supports the use of two specifications: Datastore API and Async API. For Datastore API the schema format
+supported is JSON, while for Async API is AVRO.
+
+### Systems
+
+The 'platform' field within the 'promises' field of the port in the descriptor is used to extract the name and system
+technology of the system to be created in Blindata.
+
+| Data product descriptor field | Blindata Field | Mandatory |
+|-------------------------------|----------------|-----------|
+| `promises.platform`           | `system.name`  | -         |
+
+To extract the name and technology for association, two regular expressions must be defined within the Blindata
+configurations, as shown below.
+
+```yaml
+blindata:
+  systemNameRegex: optional regex to extract system name from schema (value optional)
+  systemTechnologyRegex: optional regex to extract system technology from schema (value optional)
+```
+
+### DatastoreApi
+
+##### Physical Entity
+
+This section describes the mapping of schema annotations to physical entity properties within the system. Physical entities represent the main data structures, such as tables or views, in the data store.
+Physical entities are mappen from "schema.content" in case of single entity "schema.content.entities" in case of multiple
+entities.
+
+| Schema Annotation      | Physical Entity Property | Description                                                         | Mandatory |
+|------------------------|--------------------------|---------------------------------------------------------------------|-----------|
+| `schema.name`          | name                     | The name of the physical entity.                                    | ✔️        |
+| `schema.physicalType`  | tableType                | Specifies the type of the table (e.g., TABLE, VIEW).                | -         |
+| `schema.description`   | description              | A detailed description of the physical entity.                      | -         |
+| `schema.status`        | add.prop                 | The current status of the physical entity (e.g., active, inactive). | -         |
+| `schema.tags`          | add.prop                 | Tags for categorizing the physical entity.                          | -         |
+| `schema.domain`        | add.prop                 | The domain to which the physical entity belongs.                    | -         |
+| `schema.contactPoints` | add.prop                 | Contact information for the physical entity.                        | -         |
+| `schema.scope`         | add.prop                 | The scope of the physical entity within the system.                 | -         |
+| `schema.externalDocs`  | add.prop                 | Links to external documentation related to the physical entity.     | -         |
+
+#### Physical Field
```

Comment:
Modificare in "From JSONSchema to Physical Fields".
Aggiornare l'indice.
Verificare il livello del titolo (4 o 5 #? bisogna rimanere coerenti con il precedente e i successivi)